spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-local/src/main/java/org/springframework/cloud/dataflow/module/deployer/local/LocalModuleDeployer.java

```diff
@@ -86,7 +86,8 @@ public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		}
 		args.put("endpoints.shutdown.enabled", "true");
 
-		ModuleLaunchRequest moduleLaunchRequest = new ModuleLaunchRequest(module, args);
+		ModuleLaunchRequest moduleLaunchRequest = new ModuleLaunchRequest(String.format("%s.%s",
```

Comment:
Is there a corresponding change in the module launcher project that expects the module name to be in this form? Unfortunately there is no JavaDoc in ModuleLaunchRequest that indicates the expected format of the `module` string. Perhaps that could be addressed in a separate PR?